docs/reference/run.md

```diff
@@ -1365,9 +1365,11 @@ Similarly the operator can set the **hostname** with `-h`.
 
 ### TMPFS (mount tmpfs filesystems)
 
-    --tmpfs=[]: Create a tmpfs mount with: container-dir[:<options>], where the options are identical to the Linux `mount -t tmpfs -o` command.
+    --tmpfs=[]: Create a tmpfs mount with: container-dir[:<options>], where the
+    options are identical to the Linux `mount -t tmpfs -o` command.
```

Comment:
I think this needs to be all on the same line, unfortunately. Maybe indent the second line? I don't know what happens, but it looks odd:
![Uploading Screen Shot 2016-01-26 at 20.30.21.png…]()